code/game/objects/items/storage/backpack.dm

```diff
@@ -68,7 +68,7 @@
 
 /obj/item/storage/backpack/santabag
 	name = "Santa's Gift Bag"
-	desc = "Space Santa uses this to deliver presents to all the nice children in space in Christmas! Wow, it's pretty big!"
+	desc = "Space Santa uses this to deliver presents to all the nice children in space at Christmas! Wow, it's pretty big!"
```

Comment:
I'd say `for christmas` tbh

Reply:
or just `to deliver christmas presents to all the nice children in space`, flows better IMO